aes/Cargo.toml

```diff
@@ -17,13 +17,15 @@ categories = ["cryptography", "no-std"]
 [dependencies]
 cfg-if = "1"
 cipher = "=0.3.0-pre"
+cpuid-bool = { version = "0.1", optional = true }
```

Comment:
Ideally `cpuid-bool` should go to target-specific feature. But I don't remember how exactly optional target-specific features behave...